### ISSUE TITLE
Compile failure on macOS platform

### ISSUE THREAD

**Issue by cschol** [gibbonjoyeux/VCV-Biset#3]
Compile failure on macOS platform with Rack Plugin Toolchain:

```
src/Tracker/TrackerDisplay-draw_pattern.cpp:304:5: error: use of undeclared identifier 'itoa'; did you mean 'itoaw'?                                                                                                                                                                                                   
                                itoa(note->pitch / 12, str, 10);                                                                                                                                                                                                                                                       
                                ^~~~                                                                                                                                                                                                                                                                                   
                                itoaw                                                                                                                                                                                                                                                                                  
src/Tracker/../Tracker/Tracker.hpp:575:6: note: 'itoaw' declared here                                                                                      
void itoaw(char *str, int number, int width);                                                                                                              
     ^                                                                                                                                                                                                                                                                                                                 
src/Tracker/TrackerDisplay-draw_pattern.cpp:304:10: error: cannot initialize a parameter of type 'char *' with an rvalue of type 'int'                     
                                itoa(note->pitch / 12, str, 10);                                                                                           
                                     ^~~~~~~~~~~~~~~~                                                                                                                                                                                                                                                                  
src/Tracker/../Tracker/Tracker.hpp:575:18: note: passing argument to parameter 'str' here                                                                  
void itoaw(char *str, int number, int width);                                                                                                              
                 ^                                                                               
```

Also:

```
src/Tracker/../components.hpp:518:23: warning: suggest braces around initialization of subobject [-Wmissing-braces]
                this->color_font = {0, 0, 0, 1};
                                    ^~~~~~~~~~
                                    {         }
In file included from src/Tracker/Tracker.cpp:2:
In file included from src/Tracker/../Tracker/Tracker.hpp:5:
In file included from src/Tracker/../Segfault/../plugin.hpp:3:
In file included from /home/build/rack-plugin-toolchain/Rack-SDK-mac-x64/include/rack.hpp:35:
In file included from /home/build/rack-plugin-toolchain/Rack-SDK-mac-x64/include/helpers.hpp:7:
In file included from /home/build/rack-plugin-toolchain/Rack-SDK-mac-x64/include/app/PortWidget.hpp:5:
/home/build/rack-plugin-toolchain/Rack-SDK-mac-x64/include/engine/Module.hpp:157:23: error: 'rack::engine::SwitchQuantity::labels' is not a member of class 'ParamQuantityMode'
                sq->SwitchQuantity::labels = labels;
                    ~~~~~~~~~~~~~~~~^
```

**Comment by gibbonjoyeux** [comment]
Version: 2.0.5
Fixed hash: 7839370586fba46020357a0b066f19ca3aabdf91

**Comment by cschol** [comment]
Confirmed.
